cfpb/cfgov-refresh
# Fixing Travis with a new logo.svg

I tested different versions of gulp-imagemin, but there's just something wrong with our old logo.svg. I replaced it with a different file.

## Test 
- Run Gulp Build
- Check the logo on the homepage

## Review
- @jimmynotjim 
- @anselmbradford 
- @sebworks
